### ISSUE TITLE
How to finetune with size 448?

### ISSUE THREAD

**Issue by qinliuliuqin** [facebookresearch/mae#77]
Thanks for sharing the code and pre-trained models.
In Section 4.2, the authors mentioned: 
```
"By fine-tuning with a 448 size, we achieve 87.8% accuracy, using only IN1K data. "
```
The model is pre-trained with a size 224, how to fine-tune it with size 448?
Does anyone know the details?

**Comment by qinliuliuqin** [comment]
I figured it out after reading the code. Just interpolate the positional encoding.

**Comment by Dongshengjiang** [comment]
+1

**Comment by liuxingbin** [comment]
> I figured it out after reading the code. Just interpolate the positional encoding.

Hi!
When I finetune with size 448. I always get Nan in 30/32-epoch.
The learning rate I use is 5e-4 and  I try to clipgrad with 3.0.

Did you have something unusual in your parameter?
